python/GafferUI/SpreadsheetUI/_LinkedScrollBar.py

```diff
@@ -91,24 +91,47 @@ def __init__( self, orientation, scrolledContainers, **kw ) :
 			scrollBar.rangeChanged.connect( Gaffer.WeakMethod( self.__rangeChanged ) )
 			scrollBar.stepsChanged.connect( Gaffer.WeakMethod( self.__stepsChanged ) )
 
+		self.isUpdating = False
+
 	def __valueChanged( self, value ) :
 
+		if self.isUpdating :
+			return
+
+		self.isUpdating = True
+
 		for scrollBar in self.__scrollBars :
 			scrollBar.setValue( value )
 
+		self.isUpdating = False
```

Comment:
I don't know how likely an exception is in `setValue()`, but I do think it's worth protecting against that leaving us in a bad state where we'll ignore the next update :

```
try :
    self.__isUpdating = True
    for scrollBar in self.__scrollBars :
         scrollBar.setValue( value )
finally :
    self.__isUpdating = False
```

Reply:
Fixed and squashed into https://github.com/GafferHQ/gaffer/pull/5507/commits/d4b6571097e815b598b78142c73d1ea2d5d2f3e9